service/create/pki.go

```diff
@@ -0,0 +1,119 @@
+package create
+
+import (
+	"strings"
+
+	microerror "github.com/giantswarm/microkit/error"
+
+	"github.com/giantswarm/certctl/service/pki"
+	"github.com/giantswarm/certctl/service/token"
+)
+
+const (
+	// TODO Choose a sensible value and move to ConfigMap for V1
+	CertificateAuthorityTTL = "1440h"
```

Comment:
IMO the CA should never expire on its own. We should only actively renew it in case of security breaches. This should be super rare. What we need to renew frequently are the self signed certificates for a cluster, but this is a different story we should not care about that much right now. Anyway we should have decent default values for nearly all of the configuration the microservice cares about. So this should then be in a configmap like we have it in e.g. kubernetesd and other microservices. So lets not put this in constants here. WDYT?